proto/minder/v1/minder.proto

```diff
@@ -1253,7 +1283,12 @@ message VerifyProviderCredentialRequest {
     Context context = 1;
 
     // enrollment_nonce is the state parameter returned when enrolling the provider
-    string enrollment_nonce = 2;
+    string enrollment_nonce = 2 [
+        (buf.validate.field).string = {
+            pattern: "^[A-Za-z0-9_-]+$",
```

Comment:
Does `-` work at the end of a character class with RE2?

Reply:
Apparently this does work: https://go.dev/play/p/KlEhC41L1VD

Reply:
Yeah, I was just trying the same thing and it seemed to work alright no matter if it's put in the beginning or the end 👍